rspace/src/main/scala/coop/rchain/rspace/channelStore/instances/ChannelStoreImpl.scala

```diff
@@ -65,7 +65,7 @@ object ChannelStoreImpl {
           joinHash = hashJoinsChannel(channel, sc)
         } yield (eventKey, DataJoinHash(dataHash, joinHash))
 
-      channels.toList.traverse(convert).flatMap(store.put)
+      channels.toList.traverse(convert).flatMap(kvs => store.put(kvs.sortBy({ case (k, _) => k })))
```

Comment:
@tgrospic is this what you meant by sorting?

Reply:
I wonder, should we make sorting default on `put` API method of KVStore?

Reply:
This will not be the same sorting as LMDB does internally because it uses lexical comparison.

```scala
// Get lexical Ordering for ByteString
implicit val bsComparator: Comparator[ByteString] = ByteString.unsignedLexicographicalComparator()
// ... then use key as ByteString for sorting
...flatMap(kvs => store.put(kvs.sortBy({ case (k, _) => k.toByteString })))
```

Reply:
Also, I'm assuming that ByteString lexical comparison is the same as in LMDB.

Reply:
I see, thanks

Reply:
I did not find that this faster compared to insert without order. Maybe it requires more investigation, but this particular trick does not help much.

Reply:
This trick did not make any observable difference, might be it require more investigation.